<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongAddingDialog</class>
 <widget class="QDialog" name="SongAddingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter the song name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5" colspan="2">
    <widget class="QLineEdit" name="songNameLineEdit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Enter the name of the song text file:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>162</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QLineEdit" name="songTextLineEdit"/>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="browseSongTextPushButton">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Enter the name of the song tabulature file:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>125</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QLineEdit" name="songTabLineEdit"/>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="browseSongTabPushButton">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Enter the name of the song audio file:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>162</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="6">
    <widget class="QLineEdit" name="songAudiofileLineEdit"/>
   </item>
   <item row="6" column="6">
    <widget class="QPushButton" name="browseSongAudiofilePushButton">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Enter the name of the song translation file name:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="6">
    <widget class="QLineEdit" name="songTranslationLineEdit"/>
   </item>
   <item row="8" column="6">
    <widget class="QPushButton" name="browseSongTranslationPushButton">
     <property name="text">
      <string>Browse..</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Enter the singer name:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>126</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="4" colspan="3">
    <widget class="QLineEdit" name="singerNameLineEdit"/>
   </item>
   <item row="10" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SongAddingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SongAddingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
